authentication/views.py

```diff
@@ -32,7 +34,7 @@ def get_redirect_url(request):
     )
 
 
-class CustomLogoutView(views.LogoutView):
+class CustomLogoutView(View):
```

Comment:
Changed to a generic View to avoid deprecation errors (LoginView, LogoutView cannot be used for get requests starting with Django 5.0)